plugins/labhub.py

```diff
@@ -1,7 +1,105 @@
-from errbot import BotPlugin
+import json
+import os
+import re
 
+import github3
 
-class Labhub(BotPlugin):
-    """
-    GitHub and GitLab utilities
-    """
+from errbot import BotPlugin, re_botcmd
+
+
+class LabHub(BotPlugin):
+    """GitHub and GitLab utilities"""  # Ignore QuotesBear
+
+    def activate(self):
+        super().activate()
+
+        with open(os.path.join(os.environ.get('COBOT_ROOT', os.getcwd()),
+                               'plugins/LabHub.json')) as f:
+            config = f.read()
+
+        config = json.loads(config)
+        self.INVITE_SUCCESS = config['INVITE_SUCCESS']
+        self.ORG_USERNAME = config['ORG_USERNAME']
+
+        self.GH = LabHub.get_gh_obj(os.environ.get('GH_TOKEN'))
+        self.ORG = LabHub.get_org(self.GH, self.ORG_USERNAME)
+        self.TEAMS = LabHub.get_teams(self.ORG)
+
+    @staticmethod
+    def get_gh_obj(token):
+        """
+        :param token:   github token
+        :returns:       ``github3.GitHub`` object
+        """
+        return github3.login(token=token)
+
+    @staticmethod
+    def get_org(gh, org_name):
+        """
+        :param gh:          ``github3.GitHub`` object
+        :param org_name:    Name of the organization
+        :returns:           ``github3.orgs.Organization`` object
+        """
+        return gh.organization(org_name)
```

Comment:
this function is rather useless, a oneliner and only used once, I suggest inlining it